src/modules/form/components/DynamicField.tsx

```diff
@@ -232,6 +233,17 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
           </InputContainer>
         );
 
+      if (item.component === Component.Phone) {
+        return (
+          <PhoneInput
+            id={linkId}
+            value={value || ''}
+            onChange={onChangeValue}
+            {...commonInputProps}
+          />
+        );
+      }
```

Comment:
This is going to override the behavior on line `439` which we don't want.
The component naming is unfortunate, `PHONE` really renders 3 fields (number, type, note) which is more than what is needed here.

Can you add a new component type, maybe called `PHONE_NUMBER`, that just renders the `PhoneInput` (not the `PhoneInputGroup`?

Reply:
Sounds good!

I think it doesn't override the behavior on line 439, because that's inside of a `case ItemType.Object:` block, while this is inside of a `case ItemType.String:` block. Am I right to think that means they don't conflict?

Either way, I could still see that it would be desirable to have another way to differentiate these two. I'm happy to make a new `PHONE_NUMBER` component if you think that is the clearest way.

Reply:
With the import statement updated to pull in a real `PhoneInput` component and not a `PhoneInputGroup` deceptively renamed (see my comment [here](https://github.com/greenriver/hmis-frontend/pull/639/files#r1481842844)), this is now working correctly, aka just showing an input text box with phone validation rather than the extra dropdown and notes field.

The MultiPhone input component also still works as expected, which I think confirms that this doesn't conflict with line 439. That means the current behavior is as follows:
- if type is STRING and component is PHONE, then show a single phone input text box
- if type is OBJECT and component is PHONE, then show a multi-phone input where each phone number also has a type and notes box.

I am still happy to make a new component type to differentiate these more clearly, but I think it may not be necessary. Let me know your preference!